src/main/java/uk/gov/hmcts/reform/sandl/snlevents/actions/listingrequest/UpdateListingRequestAction.java

```diff
@@ -90,34 +77,52 @@ public void getAndValidateEntities() {
         if (hearing == null) {
             throw new EntityNotFoundException("Hearing not found");
         }
+
+        getHearingPartsWithStatus(hearing.getStatus().getStatus());
+        getSessions();
+
+        if (!hearing.isMultiSession() && updateListingRequest.getNumberOfSessions() > 1) {
+            throw new SnlEventsException("Single-session hearings cannot have more than 2 sessions!");
+        }
+
+        if (hearing.isMultiSession() && updateListingRequest.getNumberOfSessions() < 2) {
+            throw new SnlEventsException("Multi-session hearings cannot have less than 2 sessions!");
+        }
+
+        if (hearing.getStatus().getStatus().equals(Status.Listed)) {
+            if (!OffsetDateTime.now().isBefore(hearingParts.get(0).getSession().getStart())) {
```

Comment:
how do you know that hearingParts.get(0).getSession().getStart() would give you the earliest start time?

Reply:
I must've accidentally removed that part when refactoring >.< nice find!